src/platform/platform_linux.cpp

```diff
@@ -81,12 +82,12 @@ std::string mp::platform::default_server_address()
 {
     std::string base_dir;
 
-    if (mu::is_snap())
+    try
     {
         // if Snap, client and daemon can both access $SNAP_COMMON so can put socket there
         base_dir = mu::snap_common_dir().toStdString();
```

Comment:
Another particular case of the behavior change I mentioned.

Reply:
Yep